doc/sources/installation/installation-rpi.rst

```diff
@@ -6,50 +6,134 @@ Installation on Raspberry Pi
 You can install Kivy manually (recommended), or you can download and boot KivyPie on the
```

Comment:
Can you please remove any reference to KivyPie? I don't think we support this anymore so it's better to simply have the direct installation instructions.

Reply:
Fixed in: 6d2ceb51de88ac5830b67cee667f03c28eda7e14